flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/Dispatcher.java

```diff
@@ -1041,6 +1041,9 @@ protected CleanupJobState jobReachedTerminalState(ExecutionGraphInfo executionGr
                             new JobResultEntry(
                                     JobResult.createFrom(
                                             executionGraphInfo.getArchivedExecutionGraph())));
+                    log.info(
+                            "Job {} has been marked as dirty after reaching a terminal state.",
```

Comment:
Will this not confuse the hell out of users?
Couldn't it be rephrased to something more high-level like "registering job for clean up"?

Reply:
I guess, you have a point. I like your more generic version 👍